### PR TITLE
Added documentation for Flat Format

### PR DESCRIPTION
**Pull request by sidharthramesh**
With some minor changes to broken links, spelling and grammar (via Grammarly)

### CODE REVIEW

File: source/02_getting_started/02_data_models/index.rst [hunk 1]
```diff
@@ -34,8 +34,8 @@ on its lifecycle status (published, draft etc.). Choose to get the latest publis
 as export format. Clicking **Bulk Export** will then download a zip folder containing all Archetypes meeting the criteria.   
 
 Next, install the `Template Designer <http://downloads.oceaninformatics.com/downloads/TemplateDesigner/>`_. The process should
-be straight forward (At least on Windows). Alternatively, the `ADL Designer <https://ehrscape.marand.si/designerv2/>`_
-can also be used to create Templates. However, you will need to upload your files or provide them through a git instance.
+be straight forward (At least on Windows). Alternatively, the `ADL Designer <https://tools.openehr.org/designer/>`_
+can also be used to create Templates by following `this guide <https://openehr.atlassian.net/wiki/spaces/healthmod/pages/415465475/Archetype+Designer+-+template+building+manual>`_
```
Comment (jakesmolka) [inline source/02_getting_started/02_data_models/index.rst]: Can you add punctuation at the end here, i.e. `.`?

File: source/02_getting_started/05_load_data/index.rst [hunk 2]
```diff
@@ -89,5 +93,60 @@ Finally, the composition can be sent to the openEHR server:
...
+obtained by concatenating the :code:`id` of each level delimited by a :code:`/`. The last segment is the suffix and uses :code:`|` as a delimiter. 
+
+For example, in the above image all the :code:`id` to be concatenated are highlighted in red.
+
+So the paths built from the above example would look like:
+
+:code:`vital_signs/body_temperature/any_event/temperature|magnitude`
+:code:`vital_signs/body_temperature/any_event/temperature|unit`
+
+The value of these above keys would the actual data. Representing this in JSON would look like:
```
Comment (jakesmolka) [inline source/02_getting_started/05_load_data/index.rst]: Did you meant "would **be** the"?

File: source/02_getting_started/05_load_data/index.rst [hunk 3]
```diff
@@ -89,5 +93,60 @@ Finally, the composition can be sent to the openEHR server:
...
+                "vital_signs/body_temperature/any_event/temperature|unit": "°C"
+                }
+
+However, since the cardinality of the :code:`body_temperature` and :code:`any_event` elements are :code:`-1` it means that
+the composition can have an infinite number of :code:`body_temperature` and :code:`body_temperature` recorded in the same composition.
+To resolve this, we have to index the path like so:
+:code:`vital_signs/body_temperature:0/any_event:0/temperature|magnitude`
+:code:`vital_signs/body_temperature:0/any_event:0/temperature|unit`
+
+With these paths, and more context data, a composition with multiple recordings of body temperature will look like
```
Comment (jakesmolka) [inline source/02_getting_started/05_load_data/index.rst]: Can you add `:` at the end of the line for consistency?

**Review by jakesmolka — CHANGES_REQUESTED** [review]
Thank you @sidharthramesh for this addition to our docs! :slightly_smiling_face: 
Can you take a quick look at my inline comments? I'll merge this PR afterwards.

File: source/02_getting_started/05_load_data/index.rst (hunk 2 above)
Comment (sidharthramesh) [reply source/02_getting_started/05_load_data/index.rst]: Yes. My bad.

File: source/02_getting_started/05_load_data/index.rst (hunk 3 above)
Comment (sidharthramesh) [reply source/02_getting_started/05_load_data/index.rst]: Done

**Review by jakesmolka — APPROVED** [review]
Great, thanks again @sidharthramesh!

**Pull request merged**